src/bin/tikv-server.rs

```diff
@@ -953,6 +953,9 @@ fn run_raft_server(pd_client: RpcClient,
     // Stop.
     server.stop().unwrap_or_else(|err| exit_with_err(format!("{:?}", err)));
     node.stop().unwrap_or_else(|err| exit_with_err(format!("{:?}", err)));
+    if let Some(Err(e)) = worker.stop().map(|h| h.join()) {
+        info!("ignore failure when stopping resolver: {:?}", e);
```

Comment:
It's unnecessary, panic hook will exit the process. And actually we don't care whether the worker exit successfully, hence just output an info log, though this line will not be visited in current case.